src/shared-components/Recurrence/RecurrenceEndOptionsSection.module.css

```
@@ -1,7 +1,46 @@
+  margin-top: var(--space-1);
+  margin-bottom: var(--space-1);
+  display: flex;
+  align-items: center;
+}
+
+.radioLabel {
+  display: inline-block;
+  margin-right: var(--space-6);
+}
+
+.radioLabel :global(.form-check-input) {
```

Stylelint (17.2.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)